script.js

```diff
@@ -63,17 +63,18 @@ function init() {
         return new Set(users);
     };
 
-    ChatDialogue.prototype.receivedPrivateMessage = function(a) {
-        if (a.data.success) {
-            var b = this;
-            this._holodeck.filterIncomingMessage(a.data.message, function(c) {
-                b.displayUnsanitizedMessage(a.data.from, c + '&nbsp; (<a class="reply_link" onclick="if (!event.ctrlKey) { holodeck.insertPrivateMessagePrefixFor(\'' + a.data.from + '\');} else { holodeck.showConversationWith(\'' + a.data.from.toLowerCase() + '\') };return false;" href="#">reply</a>)', {
-                    "class": "whisper received_whisper"
-                }, {
-                    whisper: !0
-                });
-            });
-        } else this.kongBotMessage(a.data.to + " cannot be reached. Please try again later.");
+    ChatDialogue.MESSAGE_TEMPLATE.evaluateAntesDeWF=ChatDialogue.MESSAGE_TEMPLATE.evaluate;
+
+    ChatDialogue.MESSAGE_TEMPLATE.evaluate=function(a){
+        if(a.message) a.message=a.message.replace('<a class="reply_link" onclick="holodeck.insertPrivateMessagePrefixFor(\''+a.username+'\');return false;" href="#">reply</a>', '<a class="reply_link" onclick="if(!event.ctrlKey) holodeck.insertPrivateMessagePrefixFor(\''+a.username+'\'); else holodeck.showConversationWith(\''+a.username+'\'); return false;" href="#">reply</a>');
+        return ChatDialogue.MESSAGE_TEMPLATE.evaluateAntesDeWF(a);
+    };
+
+    ChatDialogue.prototype.sendPrivateMessageAntesDeWF=ChatDialogue.prototype.sendPrivateMessage;
+
+    ChatDialogue.prototype.sendPrivateMessage=function(a, b){
+        this.sendPrivateMessageAntesDeWF(a, b);
+        for(var room in this._holodeck._chat_window._rooms._object) if(this._holodeck._chat_window._rooms._object[room]._chat_dialogue!=this) this._holodeck._chat_window._rooms._object[room]._chat_dialogue.displayMessage(a, b, {"class": "whisper sent_whisper"}, {"private": !0});
```

Comment:
Me encantó como lo solucionaste. :+1: 
Una cosita de estilo nomás... Esa línea del `for (...) if (...) ...` se hace ilegible al ser demasiado larga jajaja
[Te dejo algo](https://github.com/airbnb/javascript) que no es necesario que sigas cuando programás en JS pero... Se suele agradecer tener un estilo de código consistente a lo largo de todo un proyecto.